airbyte-integrations/bases/base-normalization/dbt-project-template/dbt_project.yml

```diff
@@ -35,6 +35,4 @@ quoting:
 # are materialized, and more!
 models:
   airbyte:
-    # Schema (dataset) defined in profiles.yml is concatenated with schema below for dbt's final output
```

Comment:
eh?

Reply:
this suffixed the schema/dataset with `_NORMALIZED`

Reply:
ah okay.